### ISSUE TITLE
km_retroarch_171_xtreme_5_12_18.hmod doesn't run NES games

### ISSUE THREAD

**Issue by bankaiprimera** [KMFDManic/NESC-SNESC-Modifications#124]
### Step 1: Describe your setup:

  * hakchi2 CE version: 2.30
  * retroarch used (filename): latest zip file with cores
  * hmod(s) used by the game you are having problens with (filenames): km_retroarch_171_xtreme_5_12_18.hmod
  
### Step 2: Describe the problem:
km_retroarch_171_xtreme_5_12_18.hmod doesn't run NES games. Screen stays black and needs power disconnect to turn off.
  
All other cores work great, but can't get NES games to work.. and would love a smaller core in size if possible just for NES games.

**Comment by gingerbeardman** [comment]
You seem to be using cluster hakchi 2.30

hakchi2CE current version is 1.2.5 https://github.com/TeamShinkansen/hakchi2/releases

**Comment by gingerbeardman** [comment]
cluster hakchi2 has been abandoned/discontinued

hakchi2CE is the only active one now, and KM can only guarantee his hmods will work with that version. see the release page for more info: https://github.com/KMFDManic/NESC-SNESC-Modifications/releases

I'm sure he will be along to explain why this is

**Comment by bankaiprimera** [comment]
Thanks man, seems I gotta to the entire setup-process  again..! No easy way to "switch" from cluster to CE I'm afraid :(

**Comment by gingerbeardman** [comment]
Yes, there is. http://snesclassicmods.com/how-to-switch-from-hakchi2-to-hakchi2-ce/

here's the contents of `km_retroarch_171_xtreme_5_12_18.hmod`

![screen shot 2018-05-18 at 18 32 45](https://user-images.githubusercontent.com/49612/40249243-40e41cfa-5aca-11e8-970b-d77ce7360ed5.png)

**Comment by bankaiprimera** [comment]
Thanks man, gonna check it out!

**Comment by KMFDManic** [comment]
@bankaiprimera You need hotfix, in xtras folder.

https://youtu.be/KThTOZidDVM

Also, /bin/fceumm and/or /bin/nestopia

**Comment by bankaiprimera** [comment]
@KMFDManic so you don’t think it’s necessary to change to CE..? Just apply hotfix and/or use nestopia?

**Comment by KMFDManic** [comment]
@bankaiprimera I always help people choose their own path, if need be.  Anything before Hakchi 2.17 would not work as well with the RetroArchs/Cores in my set.  But, 2.17-2.21f, would work as long as the hotfix and appropriate command lines are used.  Hakchi2 CE is worked on, actively, daily...And, as a result, you may find it most optimal to try to get up to speed, as it has excellent USB-HOST support, etc:)  Whichever way you decide to go, just let me know:)  Definitely do not use the old USE NESTOPIA HMOD!  Stick to command lines, only:)

**Comment by bankaiprimera** [comment]
Thanks for all the help, I switched over to the latest Hakchi2 CE with your core-set and it works great! :)
Only thing I'm struggling with is making a correct Mortal Kombat 1 .rpm sheet for mame2000. Can't seem to figure out the the arcade to joypad configuration.

**Comment by KMFDManic** [comment]
Yes, I thought I had told you about the MAME 2000 Control Update:)  You should be good to go!  Also, you may want to run MK on MAME 2003 Xtreme, since it has the truly amazing Custom OST now:)  Not sure if you got to see that in action!

**Comment by bankaiprimera** [comment]
Good stuff @kmfdmanic! Curious to see how MK runs compared to mame2000. The latter still has some audio crackling I can’t get rid of. Maybe xtreme2003 can iron that out. 
Unfortunately I don’t have a USBhost installed (yet), so OST isn’t an option for me. But awesome you menaged to do so!

**Comment by KMFDManic** [comment]
@bankaiprimera One of a half dozen videos I have done showing it in action.  You CAN still run this on NAND, if you wanted to.  It is worth doing for a few days:)  I keep it going permanently, but am running Samples via USB-HOST:)

https://www.youtube.com/watch?v=FDPJDKRyw9Y&t=1s

**Comment by bankaiprimera** [comment]
@KMFDmanic lots of games work beautifully on the mame2003 Xtreme core, but Mortal Kombat does have audio skips and crackling unfortunately.. I’m running the mk.zip rom with all default RA settings. Any tips on getting it sped up without audio glitches?

**Comment by KMFDManic** [comment]
@bankaiprimera I would recommend trying to run mk1a1.zip (which is the best running MK Variant) with combination of MAME 2003 Xtreme Core/RetroArch:)  Keep me posted!

**Comment by bankaiprimera** [comment]
@KMFDManic thanks for the advice man! using mk1a1.zip made a HUGE difference in performance; no more jittery audio and framerate is solid too! Great stuff. Currently I'm just using the mame2003 core for all my arcade needs; CPS1, CPS2, NEOGEO, Konami stuff like Sunset Riders and of course Mortal Kombat. No need for FBA of MAME2000 anymore :)

**Comment by KMFDManic** [comment]
Not only that @bankaiprimera But, Midway/Bally+Vector Games now work with MAME 2003 Xtreme/RetroArch Xtreme Combination:)  Tapper, Journey, and so on:)

If you want to run the Custom OST for MK, simply uninstall Cores/RetroArch, and do a more limited setup, and you should have some fun running MK for a bit:)  I did a NAND Tutorial for it, as well.

https://www.youtube.com/watch?v=t9gGxNi1zhE
